### ISSUE TITLE
I think improvements to the lightbox are necessary.

### ISSUE THREAD

**Issue by styromaniac** [HelloZeroNet/ZeroMe#143]
On mobile devices, in portrait mode, pictures get cut off on the sides if they're of wide aspect ratio. In landscape, images are bound at the very top of the page instead of within the viewport.

Can't you take an already refined lightbox like Lity by Jan Sorgalla and retrofit it for ZeroMe? I don't know how to make it work for ZeroMe since I don't know JavaScript, but it works great for Kave.

TL;DR: ZeroMe needs a better lightbox. I suggest one that's well-tested.

**Comment by styromaniac** [comment]
Never mind. IDK why my clone is goofy and OG ZeroMe isn't.

I would still like a highly refined lightbox with ZeroMe though.
